addons/map/XEH_postInitClient.sqf

```diff
@@ -97,35 +78,33 @@ call FUNC(determineZoom);
 
     //illumination settings
     if (GVAR(mapIllumination)) then {
-        GVAR(flashlightInUse) = "";
-        GVAR(glow) = objNull;
-
         ["ace_playerInventoryChanged", {
```

Comment:
As far as I understand, this EH is meant to remove the current glow if the unit suddenly lost  it's flashlight. I'd appreciate if you could add a comment stating that.